### DISCUSSION TITLE
What novel begins with this phrase?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#598] [Galaxy Trivia] [answered]
*"In my younger and more vulnerable years my father gave me some advice that I’ve been turning over in my mind ever since"*

**Comment by peterbaccaro** [comment]
The novel that begins with the phrase "In my younger and more vulnerable years my father gave me some advice that I’ve been turning over in my mind ever since" is "**The Great Gatsby**" by F. Scott Fitzgerald. This iconic novel, first published in 1925, is a classic work of American literature and is widely studied in schools and universities.

**Comment by thuongtruong1009** [comment, marked as answer]
The novel that begins with the phrase "In my younger and more vulnerable years my father gave me some advice that I’ve been turning over in my mind ever since" is "The Great Gatsby" by F. Scott Fitzgerald.

**Comment by mo-cherif** [comment]
The novel that begins with the phrase "In my younger and more vulnerable years my father gave me some advice that I’ve been turning over in my mind ever since" is "The Great Gatsby" by F. Scott Fitzgerald.
